src/components/Modal/Modal.stories.jsx

```diff
@@ -10,6 +10,18 @@ import store from '../../store/index';
 // components
 import Modal from './index';
 
+global.FB = {};
+global.gapi = {};
+global.FB.login = jest.fn();
+global.FB.getLoginStatus = (cb) => {
+  const res = {
+    status: 'connected',
+    authResponse: { access_token: 'theAccessToken' }
+  };
+  cb(res);
+};
+global.gapi.load = () => {};
+
 storiesOf('Modal', module)
   .add('default signup view', () => (
     <Provider store={store}>
```

Comment:
Remove the provider and write the story for the main component not the connected one

Reply:
The child components are connected to redux and so need this to be wrapped in a mock store

Reply:
This means the component is not a reusable one if it has connected children. Please remove the story or make the component reusable.